src/seth/libexec/seth/seth-slot

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+### seth-slot -- Prints the slot number for the specified mapping type and input data
```

Comment:
slot seems too generic for what this actually is imo. wdyt about `mapping-slot` or somethin?

Reply:
I vote `seth index`

Reply:
i like it!

Reply:
`seth index` it will be! Sorry for the delay here, hoping to finish this PR up this week